ibm_catalog.json

```diff
@@ -653,7 +653,14 @@
               "hidden": true
             },
             {
-              "key": "default_worker_pool_labels"
+              "key": "default_worker_pool_labels",
+              "hidden": true,
```

Comment:
why are you hiding this? It wasn't hidden before

Reply:
Made the change.